Dockerfile

```diff
@@ -22,8 +22,10 @@ RUN yarn install
 
 ADD Gemfile* $INSTALL_PATH
 RUN gem install bundler
-RUN bundle install --deployment --without development --without test
+RUN bundle config set --local deployment 'true'
```

Comment:
The deployment version run by this container throws a 500 error when you attempt to download the PDF for a SMART Health Card.

Reply:
Let's track this in [DVCI-161](https://jira.mitre.org/browse/DVCI-161).